### ISSUE TITLE
Issue between new release (6.08) and Enhanced

### ISSUE THREAD

**Issue by mrhenkz** [ElvUI-WotLK/ElvUI#767]
Hi

I updated to the latest version, 6.08, on a WotLK server and everything works fine a part from accessing the options. I did some research on my own and it seems that you cannot access the options on ElvUI if another addon hasn't been updated yet. 

This lead me to disable all addons apart from ElvUI related ones and the problem was still there. I use ElvUI, Enhanced, Addonskins and Swingbar.

When I used the command /luaerror on I was able to access the options but my portrait changed (which I'm guessing was causing the error).

Please look in to this if possible, otherwise, great job of making this for outdated game versions and still making improvements!

Best regards

**Comment by mobcer** [comment]
I have a problem with the character frame and any feature it gets from elvui_enhanced. For instance, all stat numbers vanished and the equipment manager not showing up. When I disable elvui_enhanced everything works as it should be without the enhancements.
`
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2166: attempt to index field 'Thumb' (a nil value)
Count: 1

Call Stack:
(tail call): ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2166: in function `FixHybridScrollBarSkin'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2251: in function `Initialize'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2810: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2809>
...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:155: in function <...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:155>
[string "safecall Dispatcher[1]"]:4: in function <[string "safecall Dispatcher[1]"]:4>
[C]: ?
[string "safecall Dispatcher[1]"]:13: in function `?'
...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: in function `Fire'
Interface\AddOns\ElvUI\Core\Core.lua:1081: in function `CallLoadedModule'
Interface\AddOns\ElvUI\Core\Core.lua:1106: in function `InitializeModules'
Interface\AddOns\ElvUI\Core\Core.lua:1266: in function `Initialize'
Interface\AddOns\ElvUI\Init.lua:211: in function <Interface\AddOns\ElvUI\Init.lua:210>
`
`
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2197: Usage: GameTooltip:SetText("text" [, color])
Count: 2

Call Stack:
[C]: ?
[C]: in function `SetText'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2197: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2191>
`
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:532: attempt to index global 'CharacterStatsPane' (a nil value)
Count: 2

Call Stack:
[C]: ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:532: in function `CharacterFrame_Expand'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2183: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2176>
`
`
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:501: attempt to index field '?' (a nil value)
Count: 1

Call Stack:
[C]: ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:501: in function `CharacterFrame_Collapse'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2179: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2176>
`

**Comment by mobcer** [comment]
I have a problem with the character frame and any feature it gets from elvui_enhanced. For instance, all stat numbers vanished and the equipment manager not showing up. When I disable elvui_enhanced everything works as it should be without the enhancements.

```
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2166: attempt to index field 'Thumb' (a nil value)
Count: 1

Call Stack:
(tail call): ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2166: in function `FixHybridScrollBarSkin'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2251: in function `Initialize'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2810: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2809>
...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:155: in function <...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:155>
[string "safecall Dispatcher[1]"]:4: in function <[string "safecall Dispatcher[1]"]:4>
[C]: ?
[string "safecall Dispatcher[1]"]:13: in function `?'
...ies\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: in function `Fire'
Interface\AddOns\ElvUI\Core\Core.lua:1081: in function `CallLoadedModule'
Interface\AddOns\ElvUI\Core\Core.lua:1106: in function `InitializeModules'
Interface\AddOns\ElvUI\Core\Core.lua:1266: in function `Initialize'
Interface\AddOns\ElvUI\Init.lua:211: in function <Interface\AddOns\ElvUI\Init.lua:210>

```

```

...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2197: Usage: GameTooltip:SetText("text" [, color])
Count: 2

Call Stack:
[C]: ?
[C]: in function `SetText'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2197: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2191>

``` 
```
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:532: attempt to index global 'CharacterStatsPane' (a nil value)
Count: 2

Call Stack:
[C]: ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:532: in function `CharacterFrame_Expand'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2183: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2176>

``` 

```

...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:501: attempt to index field '?' (a nil value)
Count: 1

Call Stack:
[C]: ?
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:501: in function `CharacterFrame_Collapse'
...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2179: in function <...s\ElvUI_Enhanced\Modules\Blizzard\CharacterFrame.lua:2176>

```

**Comment by Deserthor** [comment]
I had the same problem when updated to the version 6.08, and by disabling its plugins one by one, I found out that the problem was the Swingbar, so the best solution for now at least is to desactivate Swingbar until it gets fixed.

**Comment by mobcer** [comment]
> 
> 
> I had the same problem when updated to the version 6.08, and by disabling its plugins one by one, I found out that the problem was the Swingbar, so the best solution for now at least is to desactivate Swingbar until it gets fixed.

Is it working fine with Enhanced enabled for you?

**Comment by Deserthor** [comment]
> 
> 
> > I had the same problem when updated to the version 6.08, and by disabling its plugins one by one, I found out that the problem was the Swingbar, so the best solution for now at least is to desactivate Swingbar until it gets fixed.
> 
> Is it working fine with Enhanced enabled for you?

Yes, aside from the Swingbar problem, everything else works perfectly (Enhanced plugin included).
